build/universal/trending/english.yaml

```
@@ -1,60 +1,60 @@
+article3link: "https://www.freecodecamp.org/news/python-absolute-value-python-abs-tutorial/"
+article4title: "Java Switch Statement"
+article4link: "https://www.freecodecamp.org/news/java-switch-statement-how-to-use-a-switch-case-in-java/"
+article5title: "Change Font with HTML"
+article5link: "https://www.freecodecamp.org/news/how-to-change-font-with-html/"
+article6title: "502 Bad Gateway Error"
+article6link: "https://www.freecodecamp.org/news/error-502-bad-gateway-meaning/"
+article7title: "Static Keyword in Java"
+article7link: "https://www.freecodecamp.org/news/why-static-in-java-what-does-this-keyword-mean/"
+article8title: "String to Number in JS"
+article8link: "https://www.freecodecamp.org/news/javascript-convert-string-to-number-js-string-to-int-example/"
+article9title: "Python Global Variable"
+article9link: "https://www.freecodecamp.org/news/global-variable-in-python-non-local-python-variables/"
+article10title: "Python Set"
+article10link: "https://www.freecodecamp.org/news/python-set-how-to-create-sets-in-python/"
+article11title: "What is REST?"
+article11link: "https://www.freecodecamp.org/news/what-is-rest-rest-api-definition-for-beginners/"
+article12title: "What is CRUD?"
+article12link: "https://www.freecodecamp.org/news/crud-operations-explained/"
+article13title: "%.2f in Python"
+article13link: "https://www.freecodecamp.org/news/2f-in-python-what-does-it-mean/"
+article14title: "Git Reset Origin"
+article14link: "https://www.freecodecamp.org/news/git-reset-origin-how-to-reset-a-local-branch-to-remote-tracking-branch/"
+article15title: "What is Localhost?"
+article15link: "https://www.freecodecamp.org/news/what-is-localhost/"
+article16title: "JS Array.includes()"
+article16link: "https://www.freecodecamp.org/news/check-if-an-item-is-in-an-array-in-javascript-js-contains-with-array-includes/"
+article17title: "Compare Dates in JS"
+article17link: "https://www.freecodecamp.org/news/javascript-date-comparison-how-to-compare-dates-in-js/"
+article18title: "Python Split String"
+article18link: "https://www.freecodecamp.org/news/how-to-split-a-string-in-python/"
+article19title: "Python .enumerate()"
+article19link: "https://www.freecodecamp.org/news/python-enumerate-what-is-the-enumerate-function-in-python/"
+article20title: "Create a Set in Python"
+article20link: "https://www.freecodecamp.org/news/how-to-create-a-set-in-python/"
+article21title: "Remove a Dict in Linux"
+article21link: "https://www.freecodecamp.org/news/how-to-remove-a-directory-in-linux/"
+article22title: "What is Coding Used For?"
+article22link: "https://www.freecodecamp.org/news/what-is-coding-used-for/"
+article23title: "JavaScript String Format"
+article23link: "https://www.freecodecamp.org/news/javascript-string-format-how-to-format-strings-in-js/"
+article24title: "JavaScript String to Date"
+article24link: "https://www.freecodecamp.org/news/javascript-string-to-date-date-parsing-in-js/"
+article25title: "What’s a Data Analyst Do?"
```

Are we ok with special apostrophes?
```suggestion
article25title: "What's a Data Analyst Do?"
```

---

Pretty sure they've been fine in the past. Line 59 has one in the old list.

---

Either way should be fine. I just tested this locally with a quick CI build and the special apostrophe here is visible, but it still gets highlighted when searching for a regular `'`, at least in Chrome.